api/src/main/resources/application-kakao-map.yml

```
@@ -0,0 +1,6 @@
+spring:
+  config:
+    activate:
+      on-profile: kakao-map
+kakao:
+  rest-api-key: ${KAKAO_CLIENT_ID}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)